README.md

```diff
@@ -20,10 +20,10 @@ repository](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/tree
 not clone this repository into a Foxy workspace.
 
 ## License
-The majority of this repository is licensed under the BSD-3-Clause license. However, the **UR20
-meshes** are licensed under Universal Robots A/S’ Terms and Conditions for Use of Graphical Documentation. See the separate
-[LICENSE.txt](/meshes/ur20/LICENSE.txt) file for details. If you have any questions regarding the
-license or the license doesn't fit you use-case, please contact [legal@universal-robots.com](mailto:legal@universal-robots.com).
+The [UR20 meshes](ur_description/meshes/ur20) constitutes “Graphical Documentation” the use of which is subject to and governed by our “[Terms and Conditions for use of Graphical Documentation](https://www.universal-robots.com/legal/terms-and-conditions/terms_and_conditions_for_use_of_graphical_documentation.txt)”.\
```

Comment:
That sounds like it is using "lawyer language". I assume this is coming from UR's law department and it is therefore required that we use this formulation? Otherwise I would suggest to simplify the first sentence a little bit.

Reply:
yes